src/FixedDataTableBufferedRows.js

```diff
@@ -121,16 +123,17 @@ class FixedDataTableBufferedRows extends React.Component {
 
     // render each row from the buffer into the static row array
     for (let i = 0; i < this._staticRowArray.length; i++) {
-      let rowIndex = rowsToRender[i];
       // if the row doesn't exist in the buffer set, then take the previous one
-      if (rowIndex === undefined) {
-        rowIndex = this._staticRowArray[i]
-          ? this._staticRowArray[i].props.index
-          : undefined;
-        if (rowIndex === undefined) {
-          this._staticRowArray[i] = null;
-          continue;
-        }
+      const rowIndex = defaultTo(
```

Comment:
From https://github.com/schrodinger/fixed-data-table-2/pull/677 where there's an extra guard check to ensure `rowIndex` doesn't go out of bounds.